schemas/hotelSchema.js

```diff
@@ -0,0 +1,65 @@
+const Joi = require('joi')
+const mongoose = require('mongoose')
+import { Room } from './roomSchema'
+
+const hotelSchema = new mongoose.Schema({
+  hotelId: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
+  ownerId: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
+  localization: {
```

Comment:
I think it should some other schema like Address or Location and there should be e.g. country, city etc.